app/javascript/src/assets/scss/widgets/_tabs.scss

```diff
@@ -1,50 +1,54 @@
 .tabs {
-  border-width: 0;
   @include padding($zero $space-normal);
   @include border-normal-bottom;
+  border-left-width: 0;
+  border-right-width: 0;
+  border-top-width: 0;
+}
 
-  .tabs-title {
-    @include margin($zero $space-one);
-
-    &:first-child {
-      margin-left: 0;
-    }
-    &:last-child {
-      margin-right: 0;
-    }
-    &:hover,
-    &:focus{
-      a {
-        color: darken($medium-gray, 20%);
-      }
-    }
+// Tab chat type
+.tab--chat-type {
+  @include flex;
 
+  .tabs-title {
     a {
-      color: $medium-gray;
-      border-bottom: 2px solid transparent;
-      font-size: $font-size-small;
-      @include position(relative, 1px null null null);
-      @include transition(all .15s $ease-in-out-cubic);
+      font-size: $font-size-default;
+      padding-bottom: $space-slab;
+      padding-top: $space-slab;
     }
+  }
+}
+
+.tabs-title {
```

Comment:
Can we Move this `.tabs-title` back inside `.tabs` ? Or is there any specific reason why you moved it out of it? Looks all good otherwise.

Reply:
Yes, there were few errors with nesting more than 3 levels deep in codeclimate. There are two ways to fix it: 1. My solution 2. Change html and classes naming.

Reply:
@kirillarutyunov Sounds good 👍